### PR TITLE
improve installation guide

### PR DESCRIPTION
**Pull request by wmorgan**
* Point out that Helm is recommended for production installations.
* Remove multi-stage instructions.
* Wordsmithing.

Signed-off-by: William Morgan <william@buoyant.io>

### CODE REVIEW

File: linkerd.io/content/2.11/tasks/install.md [hunk 1]
```diff
@@ -8,29 +8,31 @@ aliases = [
...
+(If you're using the CLI, see the [Getting Started
+Guide](../../getting-started/) for how to install the CLI onto your local
+environment.)
 
-Note also that, once the control plane is installed, you'll need to "mesh" any
-services you want Linkerd active for. See
-[Adding Your Service](../../adding-your-service/) for how to add Linkerd's data
-plane to your services.
+Once you've installed the control plane, you may want to install additional
+which add additional features i.e `viz`, `multicluster` and `jaeger`. See
```
Comment (kflynn) [inline linkerd.io/content/2.11/tasks/install.md:24]: s/which add additional//

**Review by kflynn — CHANGES_REQUESTED** [review]
One place where some words are duplicated – looks good to me other than that, though!

File: linkerd.io/content/2.11/tasks/install.md [hunk 2]
```diff
@@ -8,29 +8,31 @@ aliases = [
...
+you want Linkerd active for. See [Adding Your Services to
+Linkerd](../../adding-your-service/) for how to do this.
 
 ## Requirements
 
-Linkerd 2.x requires a functioning Kubernetes cluster on which to run. This
-cluster may be hosted on a cloud provider or may be running locally via
-Minikube or Docker for Desktop.
+Linkerd requires a functioning Kubernetes cluster on which to run. This cluster
+may be hosted on a cloud provider or may be running locally via Minikube or
```
[the remark this answers is not in the document]
Comment (wmorgan) [reply linkerd.io/content/2.11/tasks/install.md]: Agreed. I've removed this bit entirely.

File: linkerd.io/content/2.11/tasks/install.md (hunk 1 above)
Comment (wmorgan) [reply linkerd.io/content/2.11/tasks/install.md]: Ty!

**Review by kleimkuhler — APPROVED** [review]
Overall looks good. Since these changes are made for 2.11 though we'll want to make sure 2.12 gets similar edits — different only in that 2.12 now has the `linkerd install --crds` step.

**Pull request merged**